### DISCUSSION TITLE
recover a golden smartphone Huawei p8 Gra-ul00

### DISCUSSION THREAD

**Discussion by francesco2002x** [phhusson/treble_experimentations#2589] [Q&A]
It could be created for Huawei P8 Gra-ul00 even if it has no support, I ask you please, otherwise this smartphone is a waste, it has a Hisilcon Kirin 930 with 3GB of RAM and 16GB of ROM also a port to Android 8 GO or later I would also be fine with x86-64 (i.e. the 32bit based on 64bit) from Arm obviously, so as to have a usable smartphone nowadays even if it is from 2015, I would thank you and also make a donation of money

**Comment by devrieshh** [comment]
Cant  you install a    AOSP 12.1 v416 ,those are arm32bit releases there
system-squeak-arm32_binder64-ab-vndklite-gogapps.img.xz   for example

https://github.com/phhusson/treble_experimentations/releases

**Reply by francesco2002x** [reply]
Could you explain to me how I can install it please, since it is not possible to use the gsi selection??😭😭
I have android 6

**Reply by devrieshh** [reply]
You have to upgrade at least to android 8.1 your device has to be treble check ready treble-app.
You better search for the older custom roms

**Comment by francesco2002x** [comment]
no, it's no longer needed, I got the GSI Android 11 version installed with
twrp and it works wonderfully.
and then as I said it is a waste with custom roms, with many limitations
compared to a gsi, 1 because they are stuck at 7.1.2 of lineageos, because
they cannot start the camera, 2 the GPU does not process graphics
acceleration correctly, 3 you have many impediments, your gsi is better
than stock android 11 also because everything works.
(PS. I'm Italian, I don't know English or American very well, I'm
translating from Google)
Tacks for aiuto 😜😝😝😝😝
Il 20 nov 2023 1:57 AM, "devrieshh" ***@***.***> ha scritto:
